libs/package.json

```diff
@@ -20,6 +20,7 @@
   "dependencies": {
     "@audius/hedgehog": "^1.0.8",
     "abi-decoder": "^1.2.0",
+    "ajv": "^6.12.2",
```

Comment:
AJV is 117Kb :(
https://bundlephobia.com/result?p=ajv@6.12.2


DJV is 29.7Kb
https://github.com/korzio/djv

JSON Schema is 4Kb!!!!
https://github.com/cfworker/cfworker/tree/master/packages/json-schema

Can we use either of those?

Reply:
the cfworker json schema(requires esm modules) and djv(validates parts of the schema at a time) aren't viable. i found another package(https://bundlephobia.com/result?p=jsonschema@1.2.6) but the caveat here is we would need to fix the metadata being validating it for missing properties, invalid type etc. also we should figure. out if invalid metadata raises errors or logs.